src/plugin.js

```diff
@@ -15,6 +15,7 @@ const defaults = {
   timeout: 45 * 1000,
   dismiss: defaultDismiss,
   progressDisabled: false,
+  durationTolerance: 0.5,
```

Comment:
Is .5 seconds too high of a duration tolerance? Maybe it isn't, or doesn't matter too much, but I know I've even seen HLS segments where the last one is really short (within that tolerance). Again, it might not matter, and 1 second seems a reasonable amount of time for .5 seconds, but I just wanted to make sure we don't want to start with either a too high of a tolerance and/or too low of a wait time.